code/api/src/modules/user/mutations.js

```diff
@@ -22,11 +22,48 @@ export const userSignup = {
     password: {
       name: 'password',
       type: GraphQLString
+    },
+
+    stylePreference: {
+      name: 'stylePreference',
+      type: GraphQLString
     }
   },
   resolve: create
 }
 
+// Update
+export const userUpdate = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+
+    name: {
+      name: 'name',
+      type: GraphQLString
+    },
+
+    email: {
+      name: 'email',
+      type: GraphQLString
+    },
+
+    password: {
+      name: 'password',
+      type: GraphQLString
+    },
+
+    stylePreference: {
+      name: 'stylePreference',
+      type: GraphQLString
```

Comment:
Also will need to be an array?

Reply:
Yeah, what @muydanny said, this data type will need to be an array, or we can pass you an entire string that will need to be broken up into an array on the BE; So that that @jhgould can run his 🔥 algorithm. –OR– if that doesn't pan out we can just run the style assessment algorithm on the FE. Whatever works!